1.0/docs/devguide/registering-elements.md

```diff
@@ -131,6 +128,45 @@ To use a type-extension element in markup, use the _native_ tag and add an
 <!-- legacy anchor -->
 <a id="basic-callbacks"></a>
 
+### Define an element in the main HTML document {#main-document-definitions}
+
+To define an element in your main HTML document, define the element
+from `HTMLImports.whenReady(callback)`. `callback` is invoked when 
+all imports in the document have finished loading.
+
+
+    <!DOCTYPE html>
+    <html>
+        <head>
+            <script src="bower_components/webcomponentsjs/webcomponents-lite.js">
+            </script>
+            <link rel="import" href="bower_components/polymer/polymer.html">
+            <title>Defining a Polymer Element from the Main Document</title>
+        </head>
+        <body>
+            <dom-module id="main-document-element">
+                <template>
+                    <p>
+                        Hi! I'm a Polymer element that was defined in the
+                        main document!
+                    </p>
+                </template>
+                <script>
+                    HTMLImports.whenReady(function () {
+                        Polymer({
+                            is: 'main-document-element'
+                        });
+                    });
+                </script>
+            </dom-module>
+            <main-document-element></main-document-element>
+        </body>
+    </html>
+
+Defining elements from the main document is primarily useful for small 
```

Comment:
This sentence at the end. Basically find a way to highlight that this should only be used for jsbin/testing. Otherwise, people create elements that aren't reusable :(